core/controllers/retrieve.py

```diff
@@ -0,0 +1,33 @@
+from io import BytesIO
+
+import flask
+from flask import abort, request
+from sqlalchemy import select
+
+from core.const import EXCEL_MIMETYPE
+from core.db import db
+from core.db.entities import Submission
+
+
+def retrieve():
+    """Handle the download request and return the originally submitted spreadsheet.
+
+    Select file by:
+    - UUID
+
+    :return: Flask response object containing the requested spreadsheet.
+    """
+    submission_UUID = request.args.get("submission_UUID")
+
+    query = select(Submission.submission_file, Submission.reporting_round).where(Submission.id == submission_UUID)
```

Comment:
I thought we decided to go with Submission ID rather than UUID?

Reply:
Yes, I'm swapping over to use submission_id instead